EFAuditer/obj/EFAuditer.csproj.EntityFrameworkCore.targets

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Similarly with these, I think the whole `obj` folder can be gitignored too

Reply:
Agreed - I added a gitignore file for this project, some files were still left but I hope that's because they were modified? Will see what happens when I merge...